recommender/engines/nlp_embedders/embedders.py

```diff
@@ -0,0 +1,106 @@
+# pylint: disable=expression-not-assigned, missing-function-docstring, too-few-public-methods
+
+"""External NLP based embedders"""
+
+from typing import Union, Iterable, List, Tuple
+
+import torch
+from mongoengine import QuerySet
+from pandas import DataFrame
+from torch import Tensor
+
+from recommender.engines.nlp_embedders.handlers import (
+    Handler,
+    ListOfObjectsHandler,
+    StringHandler,
+    ListOfStringsHandler,
+    ObjectHandler,
+)
+from recommender.engines.rl.utils import create_index_id_map
+
+SERVICE_EMBEDDING_DIM = "SE"
+USER_EMBEDDING_DIM = "UE"
+
+
+class Objects2tensorsEmbedder:
```

Comment:
Not necessarily, there is a bunch of possible handlers (5) and it would explode to 5 classes + abstract one while it is perfectly fine to inject handlers as a strategy and keep classes hierarchy as simple as possible.